HW_6/HW_5a.py

```diff
@@ -0,0 +1,144 @@
+            if os.path.exists(user_input):
+                input_file = open(user_input, 'r+')
+            else:
+                print("I did not find the file at, " + str(user_input) + " I'll use mine")
+                input_file = open('default_news.txt', "r")
+                parsed_text = input_file.read().split('\n\n')
+                for i in parsed_text:
+                    text = i[0:]
+                    location = 'Default location'
+                    story = News(header, text, location)
+                    story.print_to_file(story.format_text(header, text, location), 'sum_news.txt')
+            input_file.close()
+            os.remove('default_news.txt')
+        else:
+            location = input('Enter location \n:')
+            story = News(header, text, location)
+            story.print_to_file(story.format_text(header, text, location), 'sum_news.txt')
+    elif header == 'PrivatAdv':
+        if len(text) == 0:
+            user_input = input("Enter the path of your file: ")
+            if os.path.exists(user_input):
+                input_file = open(user_input, 'r+')
+            else:
+                print("I did not find the file at, " + str(user_input) + "I'll use mine")
+                input_file = open('default_advs.txt', "r")
+                parsed_text = input_file.read().split('\n\n')
+                for i in parsed_text:
+                    text = i[0:]
+                    expiration_date = datetime.date.today()
+                    advertisement = PrivatAdv(header, text, expiration_date)
+                    advertisement.print_to_file(advertisement.format_text(header, text, expiration_date),
+                                                'sum_news.txt')
+            input_file.close()
+            os.remove('default_advs.txt')
+        else:
+            expiration_date = list(datefinder.find_dates(
+                input('Till what date we should keep it?\nPlease follow DD-MM-YYYY date version:')))[0].date()
+            advertisement = PrivatAdv(header, text, expiration_date)
+            advertisement.print_to_file(advertisement.format_text(header, text, expiration_date), 'sum_news.txt')
+    elif header == 'Consider':
+        if len(text) == 0:
+            user_input = input("Enter the path of your file: ")
+            if os.path.exists(user_input):
+                input_file = open(user_input, 'r+')
+            else:
+                print("I did not find the file at, " + str(user_input) + "I'll use mine")
+                input_file = open('default_considers.txt', "r")
+                parsed_text = input_file.read().split('\n\n')
+                for i in parsed_text:
+                    text = i[0:]
+                    score = random.randint(1, 100)
+                    game = Consider(header, text, score)
+                    game.print_to_file(game.format_text(header, text, score), 'sum_news.txt')
+            input_file.close()
+            os.remove('default_considers.txt')
```

Comment:
It's extremely important here to verify that file was successfully processed before deleting.